packages/core/mesh/edge-client/src/edge-http-client.ts

```diff
@@ -259,8 +259,26 @@ export class EdgeHttpClient {
     body: UploadFunctionRequest,
     args?: EdgeHttpGetArgs,
   ): Promise<UploadFunctionResponseBody> {
+    const formData = new FormData();
+    formData.append('name', body.name ?? '');
+    formData.append('version', body.version);
+    formData.append('ownerPublicKey', body.ownerPublicKey);
+    formData.append('entryPoint', body.entryPoint);
+    for (const [filename, content] of Object.entries(body.assets)) {
+      formData.append(
```

Comment:
I'm not 100% sure this is required, but Cloudflare uses `filename` as the chunk name, so its safer to stick to that.
I believe this is also the key for the chunk when form data gets parsed on the server.

```ts
      formData.append(
        filename,
        new Blob([content as Uint8Array<ArrayBuffer>], { type: 'application/octet-stream' }),
        filename,
      );
      ```